PowerEditor/src/ScintillaComponent/GoToLineDlg.cpp

```diff
@@ -124,40 +114,26 @@ intptr_t CALLBACK GoToLineDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM)
                     return TRUE;
                 }
 
-				case IDC_RADIO_GOTOLINE :
-				case IDC_RADIO_GOTOOFFSET :
+				case IDC_RADIO_GOTOLINE:
+				case IDC_RADIO_GOTOOFFSET:
 				{
-				
-					bool isLine, isOffset;
 					if (wParam == IDC_RADIO_GOTOLINE)
 					{
-						isLine = true;
-						isOffset = false;
 						_mode = go2line;
 					}
 					else
 					{
-						isLine = false;
-						isOffset = true;
 						_mode = go2offsset;
 					}
-					::SendDlgItemMessage(_hSelf, IDC_RADIO_GOTOLINE, BM_SETCHECK, isLine, 0);
-					::SendDlgItemMessage(_hSelf, IDC_RADIO_GOTOOFFSET, BM_SETCHECK, isOffset, 0);
+
+					setChecked(IDC_RADIO_GOTOLINE, _mode == go2line);

```

Comment:
@ozone10,

Thanks again for your great work. 👍 

Just curious: is `SendDlgItemMessage()` or `setChecked()` needed here?